reactDashboard/src/pages/Regional.jsx

```diff
@@ -47,6 +47,23 @@ const Regional = ({
 
   const currentRegionCode = useRef(regionCode);
   const currentLocation = useRef(match.url);
+  const [dateRange, setDateRange] = useState("dateRange");
```

Comment:
Same here - dateRange and maxDateRange state (and the effect that sets their initial values) could be pushed down into the dataChart components - they're not needed for anything else

Reply:
moved to separate ticket